torch/_inductor/utils.py

```diff
@@ -256,11 +257,15 @@ def convert_shape_to_symint(
     from .virtualized import V
 
     return [
-        i
-        if isinstance(i, int)
-        else int(i)
-        if isinstance(i, sympy.Integer)
-        else V.graph.sizevars.shape_env.create_symintnode(i, hint=None)
+        (
```

Comment:
this is from linter, no logic change, can revert if needed

Reply:
This file is UFMT enabled, why is the linter wobbling these lines 🤔